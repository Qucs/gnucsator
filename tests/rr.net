# Qucs 0.0.19

R:R1 _net0 _net1 R="50 Ohm" Temp="26.85" Tc1="0.0" Tc2="0.0" Tnom="26.85"
VProbe:Pr1 _net0 gnd
.DC:DC1
Vdc:V1 _net0 gnd U="1 V"
R:R2 _net1 gnd R="100 Ohm"
