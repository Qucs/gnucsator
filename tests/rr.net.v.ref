
R #(.R(50),.Tc1(0.0),.Tc2(0.0),.Temp(26.85),.Tnom(26.85),.ic(NA( NA))) R1 (.p(_net0),.n(_net1));
Vprobe #() Pr1 (.p(_net0),.n(0));
Vdc #(.U(1),.Temp(NA( NA))) V1 (.p(_net0),.n(0));
R #(.R(100),.Tc1(NA( NA)),.Tc2(NA( NA)),.Temp(NA( NA)),.Tnom(NA( NA)),.ic(NA( NA))) R2 (.p(_net1),.n(0));
