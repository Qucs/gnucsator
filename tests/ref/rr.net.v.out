
R #(.R(50),.Tc1(0.0),.Tc2(0.0),.Temp(26.85),.Tnom(26.85)) R1 (.p(_net0),.n(_net1));
VProbe #() Pr1 (.p(_net0),.n(0));
Vdc #(.U(1)) V1 (.p(_net0),.n(0));
R #(.R(100)) R2 (.p(_net1),.n(0));
